templates/pesto/src/recipe-app.html

```diff
@@ -131,9 +130,6 @@
         </app-header-layout>
       </app-drawer>
 
-      <app-scroll-position
-          selected="[[_pageData.page]]"
-          reset="[[_isDetailPage(_pageData.page)]]"></app-scroll-position>
```

Comment:
Let's add some JS code to save the scroll position so if you navigate back to the list page you are at the previous scroll position.  This code will serve as a sample for users for replacing `app-scroll-position` element.